packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -573,6 +635,11 @@ export function TransferPanel() {
         return undefined
       }
 
+      // ETH transfers using SC wallets not enabled yet
+      if (isSmartContractWallet && !selectedToken) {
+        return TransferPanelMainErrorMessage.SC_WALLET_UNSUPPORTED_TOKEN
```

Comment:
Seems like this case only happens when trying an ETH transfer with a smart contract wallet, in which case let's name it `TransferPanelMainErrorMessage.SC_WALLET_ETH_TRANSFER_NOT_SUPPORTED_` or similar?